index.html

```diff
@@ -21,18 +23,198 @@
         </div>
       </nav>
     </header>
+    <!-- Headline Section -->
     <section class="intro-section">
       <h1>I’m Cristian.<br>Glad to see you!</h1>
       <p>I’m a software developer! I can help you build a product, feature, or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
       <a class="connect-btn">LET'S CONNECT</a>
       <ul class="social-icons">
-        <li><a href=""><img src="assets/twitter.svg" alt="Twitter icon" class="social-icon"></a></li>
-        <li><a href=""><img src="assets/linkedin.svg" alt="LinkedIn icon" class="social-icon"></a></li>
-        <li><a href=""><img src="assets/medium.svg" alt="Medium icon" class="social-icon"></a></li>
-        <li><a href=""><img src="assets/github.svg" alt="GitHub icon" class="social-icon"></a></li>
-        <li><a href=""><img src="assets/angel.svg" alt="Angelist icon" class="social-icon"></a></li>
+        <li><a href=""><img src="assets/twitter.svg" alt="Cristian's Twitter" class="social-icon"></a></li>
+        <li><a href=""><img src="assets/linkedin.svg" alt="Cristian's LinkedIn" class="social-icon"></a></li>
+        <li><a href=""><img src="assets/medium.svg" alt="Cristian's Medium" class="social-icon"></a></li>
+        <li><a href=""><img src="assets/github.svg" alt="Cristian's GitHub" class="social-icon"></a></li>
+        <li><a href=""><img src="assets/angel.svg" alt="Cristian's Angelist" class="social-icon"></a></li>
       </ul>
     </section>
+<!-- Projects Section -->
+<section class="project-section" aria-label="Projects">
+    <!-- 1st project card -->
+    <article class="project-card"> 
+      <div class="project-img"> 
+        <img src="assets/Snapshoot-1.svg" alt="Screenshot of Project 1">
+      </div>
+      <div class="project-content"> 
+        <h2>Tonic</h2>
+        <div class="project-info">
+          <h3>CANOPY</h3>
+          <p>Back End Dev</p>
+          <p>2015</p>
+        </div>
+        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="project-tags">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">CSS</a></li>
+          <li><a href="#">JavaScript</a></li> 
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+    </article>
```

Comment:
According to the[ project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), kindly use `Flexbox` to lay out an individual card. 😊🛠

* This gives us a lot of options to align all our elements, plus, it's responsive.